apps/worker/src/routes/auth.ts

```
@@ -1,44 +1,355 @@
+      name: name || email.split('@')[0],
+      avatar_url: null,
+    };
+  }
+
+  return user;
+};
+
+// POST /api/auth/login - Email/password or OAuth callback
+authRouter.post('/login', async (c) => {
+  try {
+    const { email, password, name, provider } = await c.req.json();
```

ESLint

[error] 'password' is assigned a value but never used.

(no-unused-vars)